tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_SUBCONUS_Ind_3km_ics_FV3GFS_lbcs_FV3GFS_suite_WoFS_v0.yaml

```diff
@@ -0,0 +1,24 @@
+metadata:
+  description: |-
+    This test is to ensure that the workflow running in community mode
+    completes successfully on the RRFS_SUBCONUS_3km grid using the WoFS_v0
+    physics suite with ICs and LBCs derived from the FV3GFS.
+user:
+  RUN_ENVIR: community
+workflow:
+  CCPP_PHYS_SUITE: FV3_WoFS_v0
+  DATE_FIRST_CYCL: '20190615'
+  DATE_LAST_CYCL: '20190615'
+  FCST_LEN_HRS: 6
+  PREEXISTING_DIR_METHOD: rename
+task_get_extrn_ics:
+  EXTRN_MDL_NAME_ICS: FV3GFS
+  FV3GFS_FILE_FMT_ICS: grib2
+  USE_USER_STAGED_EXTRN_FILES: true
+task_get_extrn_lbcs:
+  EXTRN_MDL_NAME_LBCS: FV3GFS
+  LBC_SPEC_INTVL_HRS: 3
+  FV3GFS_FILE_FMT_LBCS: grib2
+  USE_USER_STAGED_EXTRN_FILES: true
+task_run_fcst:
+  PREDEF_GRID_NAME: SUBCONUS_Ind_3km
```

Comment:
`PREDEF_GRID_NAME` should be in `workflow:` section after `CCPP_PHYS_SUITE:`, and then remove the `task_run_fcst` section.